src/NServiceBus.TransactionalSession/OpenSessionOptions.cs

```diff
@@ -10,7 +10,7 @@
     /// <remarks>
     /// The behavior of this class is exposed via extension methods.
     /// </remarks>
-    public class OpenSessionOptions : IExtendable
+    public abstract class OpenSessionOptions : IExtendable
```

Comment:
that means that persisters wouldn't have an option to perform persistence-specifc async code as part of the `Open` call right?